shortlister/controller.py

```diff
@@ -91,14 +92,14 @@ def edit_criteria_select(self, k=None):
         self.view.view_selection_options(self.current_criterion)
         self.options = {
             str(i): (self.edit_score_confirm,s)
-            for i, s in enumerate(self.current_criterion.scores)
+            for i, s in enumerate(SCORE_AND_VALUE.keys())
```

Comment:
You don't need the `.keys()` here. When you iterate (i.e. loop) over a dictionary in Python, you get each key.